server/game/cards/03_TWI/units/CloneDiveTrooper.ts

```diff
@@ -0,0 +1,25 @@
+import AbilityHelper from '../../../AbilityHelper';
+import { NonLeaderUnitCard } from '../../../core/card/NonLeaderUnitCard';
+import { AbilityType, RelativePlayer } from '../../../core/Constants';
+
+
+export default class CloneDiveTrooper extends NonLeaderUnitCard {
+    protected override getImplementationId () {
+        return {
+            id: '5445166624',
+            internalName: 'clone-dive-trooper',
+        };
+    }
+
+    public override setupCardAbilities () {
+        this.addCoordinateAbility({
+            type: AbilityType.Constant,
+            title: 'While this unit is attacking, the defender gets -2/-0.',
+            targetController: RelativePlayer.Opponent,
+            matchTarget: (card) => card.isUnit() && card.isInPlay() && card.isDefending(),
```

Comment:
Currently this would active when _any_ friendly unit is attacking, not necessarily this one. Need to add an additional condition `card.activeAttack.attacker === context.source`. Also we should add a test confirming that other friendly units don't get the effect.